sdk/maps/maps-timezone-rest/test/public/mapsTimezone.spec.ts

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder, env } from "@azure-tools/test-recorder";
+import { isNodeLike } from "@azure/core-util";
+import { createTestCredential } from "@azure-tools/test-credential";
+import { assert } from "chai";
+import { createClient, createRecorder } from "./utils/recordedClient";
+import { Context } from "mocha";
+import MapsTimezone, { isUnexpected } from "../../src";
+import { IanaIdOutput } from "../../src";
+
+
+describe("Authentication", function () {
+  let recorder: Recorder;
+
+  beforeEach(async function (this: Context) {
+    recorder = await createRecorder(this);
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("should work with Microsoft Entra ID authentication", async function () {
```

Comment:
Tests failing with record mode. Let's skip the test while we are waiting on the investigation/fix.

https://github.com/Azure/azure-sdk-for-js/issues/30997

Reply:
Please refer https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/test-utils/recorder#skipping-tests for skipping tests.

In short...
```js
if (isPlaybackMode()) this.skip();
```

Reply:
Thanks @HarshaNalluru

Reply:
Fix this https://github.com/Azure/azure-sdk-for-js/pull/30799/files#r1751283744
and all your problems with recording tests and playing them back will be resolved.